src/lib/y2network/interface_config_builders/ib.rb

```diff
@@ -13,13 +13,17 @@ def initialize
       attr_writer :ipoib_mode
 
       def ipoib_mode
-        @ipoib_mode ||= Yast::LanItems.ipoib_mode || "default"
+        @ipoib_mode ||= if [nil, ""].include?(@config["IPOIB_MODE"])
+          "default"
+        else
+          @config["IPOIB_MODE"]
+        end
       end
 
       def save
         super
 
-        Yast::LanItems.ipoib_mode = ipoib_mode == "default" ? nil : ipoib_mode
+        @config["IPOIB_MODE"] = ipoib_mode == "default" ? nil : ipoib_mode
```

Comment:
do you test this with real write? I sometimes found that transition from LanItems to NetworkInterfaces has many cave eats.

Reply:
yes ... but testing even more just now

Reply:
manually tested and seems to be ok

Reply:
Same than above, please try to document as much as possible the new classes / methods.